app/models/index-card.ts

```diff
@@ -90,15 +90,23 @@ export default class IndexCardModel extends Model {
     async getOsfModel(options?: object) {
         const identifier = this.resourceIdentifier;
         if (identifier && this.osfModelType) {
-            const guid = this.guidFromIdentifierList();
+            const guid = this.guidFromIdentifierList;
             if (guid) {
                 const osfModel = await this.store.findRecord(this.osfModelType, guid, options);
                 this.osfModel = osfModel;
             }
         }
     }
 
-    guidFromIdentifierList() {
+    get osfUrl() {
```

Comment:
confusing that there's an `osfUrl` getter (for the url to this resource) and also a `osfUrl` setting (for the base osf url) imported in this file (and used in this getter!)

Reply:
Oof, that's just bad naming here. Yea, will rename/rewrite this along with the suggestion below